### PR TITLE
Replaced OCRAD library by powerfull TesseractJS library

### PR DESCRIPTION
**Pull request by simduchaine**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -194,13 +192,16 @@
...
-        resultText = resultText.trim();
+        //Replaced OCRAD with Tesseract
+        Tesseract.recognize(ctx).then(function(result1){
+            var resultText = result1.text;
+
+            resultText = resultText.trim();
 
         //show the result
         $('blockquote p').html('&bdquo;' + resultText + '&ldquo;');
         $('blockquote footer').text('(' + resultText.length + ' characters)')
```
Comment (kdzwinel) [inline js/main.js]: OCD triggered! :) Can you please push those three lines above to the right?

**Review by kdzwinel — CHANGES_REQUESTED** [review]
Thank you so much for contributing! I left one small comment.

Since tesseract is async and takes a bit more time, we will need to add some kind of a loading animation, but I can take care of that in a separate PR 👍

File: js/main.js (hunk 1 above)
Comment (simduchaine) [reply js/main.js]: Hahaha No Problem.

**Comment by kdzwinel** [comment]
Beautiful, thanks so much for doing this 🙌

**Pull request merged**